### DISCUSSION TITLE
Need Test Sites

### DISCUSSION THREAD

**Discussion by krypterro** [ultrafunkamsterdam/undetected-chromedriver#1609] [Q&A]
I'm reworking my code and testing different solutions, I of course have access to nowsecure.nl but I either get to that site around Cloudflare or I don't. 

Are there some public sites that will show you what the server can see, so I can compare my bot results to regular Chrome from the desktop?

**Comment by jpjacobpadilla** [comment]
You could just make a simple web server and then be able to see everything.

**Comment by krypterro** [comment]
I have plenty of those, but I don't know what to parse the logs for in order to guess at what Cloudflare might be working into their algorithm.  Also, a simple web server won't come with whatever JS they are using to integrate the client-side browser. We should really have a Python Flask project for this.
